src/helpers/json.ts

```diff
@@ -0,0 +1,37 @@
+import camelCase from 'lodash/camelCase';
+import snakeCase from 'lodash/snakeCase';
+
+export type JSONValue = string | number | boolean | JSONObject | JSONArray;
+export type JSONArray = Array<JSONValue>;
+
+export interface JSONObject {
+  [key: string]: JSONValue;
+}
+
+export const keysToCamelCase = (json: JSONValue | JSONObject | JSONArray): JSONValue | JSONObject | JSONArray => {
+  if (Array.isArray(json)) {
+    return json.map((jsonObject) => keysToCamelCase(jsonObject as JSONValue));
+  } else if (json === Object(json)) {
+    const resultObject: JSONObject = {};
+    Object.keys(json).forEach((key) => {
+      resultObject[camelCase(key)] = keysToCamelCase((json as JSONObject)[key]);
+    });
+
+    return resultObject;
+  }
+  return json;
+};
+
+export const keysToSnakeCase = (json: JSONValue | JSONObject | JSONArray): JSONValue | JSONObject | JSONArray => {
+  if (Array.isArray(json)) {
+    return json.map((jsonObject) => keysToSnakeCase(jsonObject as JSONValue));
+  } else if (json === Object(json)) {
+    const resultObject: JSONObject = {};
+    Object.keys(json).forEach((key) => {
+      resultObject[snakeCase(key)] = keysToSnakeCase((json as JSONObject)[key]);
+    });
+
+    return resultObject;
+  }
+  return json;
+};
```

Comment:
these 2 functions are similarity, can we refactor them both 👍

Reply:
@nvminhtue rèactoẻd in [800f4ab](https://github.com/manh-t/react-survey/pull/46/commits/800f4aba172dd271928aa7b3f5e24bd0a60f05f4)